terraform/deployments/variables/test/apps.tfvars

```diff
@@ -9,3 +9,5 @@ mongodb_host        = "mongo-1.test.govuk-internal.digital,mongo-2.test.govuk-in
 redis_host          = "pink-backend-redis.0f3erf.ng.0001.euw1.cache.amazonaws.com"
 sentry_environment  = "test"
 router_mongodb_url  = "mongodb://router-backend-1.test.govuk-internal.digital,router-backend-2.test.govuk-internal.digital,router-backend-3.test.govuk-internal.digital/router"
+signon_db_url       = "mysql2://root:root@mysql-primary.test.govuk-internal.digital/signon"
```

Comment:
We'll need to rejig these at some point (for example to separate the hostname from the database name and probably also the credentials) but don't worry about it for now.